_sass/_imla-nama.scss

```diff
@@ -122,23 +130,25 @@ p {
 .toc-index {
     margin-bottom: 2.6rem;
     text-align: right;
-    counter-reset: toc-index-count toc-index-count-roman;
+    counter-reset: toc-index-count toc-index-count-abjad;
 
     li::before {
-        font-family: LalezarWeb, sans-serif;
+        font-family: NotoNastaliqUrdu, sans-serif; //Removed Lalezar due to lack of Urdu numeral support
+        font-weight:bold;
```

Comment:
We haven't included a bold weight of Noto Nastaliq Urdu, so the browser will use faux bold here, which is bad. We can include the bold weight, but doing so just for the numerals will be overkill (unless we subset the bold weight to include just the numerals). So either remove the bold weight and adjust the color a bit, or subset the bold font of Noto Nastaliq Urdu to include only the numerals ([fonttools](https://github.com/fonttools/fonttools) have a nice subsetter).

(Please also make this change at other places too where faux bold is being used.)

Reply:
I'll subset the bold weight but why are tff files being provided for Noto Nastaliq along with woff versions?
![image](https://user-images.githubusercontent.com/62665768/177015190-f5a714ae-8bf3-4fd0-9bc6-84e1f890e285.png)
 Should the same be done for the bold weight? And if not, should it be subset before or after converting to woff?

Reply:
Ah, you can remove the TTF file.

For subsetting, please see the [documentation](https://github.com/fonttools/fonttools/blob/97958a95e1636a45d718a49d0ebd72d7382b45c0/Lib/fontTools/subset/__init__.py#L27=) of the `pyftsubset` command within `fonttools`. You can specify the output format (woff/woff2) using the [`--flavor` option](https://github.com/fonttools/fonttools/blob/97958a95e1636a45d718a49d0ebd72d7382b45c0/Lib/fontTools/subset/__init__.py#L138=).

Reply:
Done in [453e03e](https://github.com/urduweb/imla/pull/17/commits/453e03e33069036edd4b34b4fa1af459b4cf8d07).